src/librustc_resolve/resolve_imports.rs

```diff
@@ -52,7 +52,10 @@ pub enum ImportDirectiveSubclass<'a> {
         value_result: Cell<Result<&'a NameBinding<'a>, Determinacy>>,
         type_result: Cell<Result<&'a NameBinding<'a>, Determinacy>>,
     },
-    GlobImport { is_prelude: bool },
+    GlobImport {
+        is_prelude: bool,
+        max_vis: Cell<ty::Visibility>, // The visibility of the greatest reexport.
```

Comment:
(c.f. the last paragraph in [this section](https://github.com/rust-lang/rfcs/blob/master/text/1560-name-resolution.md#re-exports-namespaces-and-visibility) of the RFC text)